server/src/test/java/io/crate/planner/operators/CollectTest.java

```diff
@@ -21,59 +21,30 @@
 
 package io.crate.planner.operators;
 
-import static io.crate.planner.optimizer.costs.PlanStatsTest.PLAN_STATS_EMPTY;
-import static org.hamcrest.Matchers.is;
-import static org.hamcrest.Matchers.nullValue;
-import static org.junit.Assert.assertThat;
+import static io.crate.testing.Asserts.assertThat;
 import static org.mockito.Mockito.mock;
 
-import java.util.List;
 import java.util.Set;
 
 import org.junit.Test;
 
 import io.crate.analyze.QueriedSelectRelation;
-import io.crate.analyze.WhereClause;
-import io.crate.analyze.relations.DocTableRelation;
 import io.crate.data.Row;
 import io.crate.execution.dsl.phases.RoutedCollectPhase;
 import io.crate.execution.dsl.projection.builder.ProjectionBuilder;
-import io.crate.expression.symbol.Symbol;
 import io.crate.planner.DependencyCarrier;
 import io.crate.planner.ExecutionPlan;
 import io.crate.planner.PlannerContext;
-import io.crate.statistics.TableStats;
 import io.crate.test.integration.CrateDummyClusterServiceUnitTest;
 import io.crate.testing.SQLExecutor;
-import io.crate.types.DataTypes;
 
 public class CollectTest extends CrateDummyClusterServiceUnitTest {
 
-    @Test
-    public void test_prune_output_of_collect_updates_estimated_row_size() throws Exception {
```

Comment:
I think this test case doesn't make sense anymore with the new plan stats system.

But I think what we can consider is adding a `averageSizePerRowInBytes` variant that takes the columns as argument.